src/docs/deployment/ios.md

```diff
@@ -53,21 +53,21 @@ Next, you'll register your app on App Store Connect:
 1. On the App Store Connect landing page, click **My Apps**.
 1. Click **+** in the top-left corner of the My Apps page, then select
    **New App**.
-1. Fill in your app details in the form that appears. In the Platforms section,
-   ensure that iOS is checked. Since Flutter does not currently support tvOS,
+1. Fill in your app details in the form that appears.
+   In the Platforms section, ensure that iOS is checked.
+   Since Flutter does not currently support tvOS,
    leave that checkbox unchecked. Click **Create**.
 1. Navigate to the application details for your app and select **App
    Information** from the sidebar.
-1. In the General Information section, select the Bundle ID you registered in
-   the preceding step.
-
-For a detailed overview, see [Add an app to your account][appstoreconnect_guide_register].
+1. In the General Information section, select the Bundle ID
+   you registered in the preceding step. For a detailed overview,
```

Comment:
Why move the second sentence into this step? It seems to apply to the whole procedure, not just this step.

Reply:
I still wonder about this one...